docs/products/flink/concepts/important_concepts.rst

```diff
@@ -0,0 +1,42 @@
+Important concepts in Apache Flink
+==================================
+
+This article gives you a brief explanation of some of the relevant terms and concepts in Apache Flink. A basic understanding of these helps you get the most out of our other Apache Flink articles.
+
+
+Event and processing times
+--------------------------
+
+Event time refers to when events actually happen, while processing time refers to when events are observed within a system. Various factors affect how events are processed, including shared hardware resources and network congestion, distributed system logic, and variances in the data throughput and order. Because of this, there can be significant differences between the event time and processing time for an event, even though ideally they would be equal.
+
+In Apache Flink, the streamed data does not always arrive in the same order as the events occurred. This means that using processing time in your applications can cause issues in system behavior. For this reason, we recommend that you use event time to process data, as it allows your applications to maintain the correct event sequence throughout the data streaming pipeline.
```

Comment:
Can we add here that processing data using event time allows you to reprocess the data at a later stage, providing always consistent results

Reply:
Good point, that's now added in ;)